test/node/bigint.test.ts

```diff
@@ -263,223 +263,225 @@ describe('BSON BigInt support', function () {
       expect(serializedMap).to.deep.equal(expectedSerialization);
     });
   });
+});
 
-  describe('EJSON.parse()', function () {
-    type ParseOptions = {
-      useBigInt64: boolean | undefined;
-      relaxed: boolean | undefined;
-    };
-    type TestTableEntry = {
-      options: ParseOptions;
-      expectedResult: BSON.Document;
+describe('EJSON.parse()', function () {
+  type ParseOptions = {
+    useBigInt64: boolean | undefined;
+    relaxed: boolean | undefined;
+  };
+  type TestTableEntry = {
+    options: ParseOptions;
+    expectedResult: BSON.Document;
+  };
+
+  // NOTE: legacy is not changed here as it does not affect the output of parsing a Long
+  const useBigInt64Values = [true, false, undefined];
+  const relaxedValues = [true, false, undefined];
+  const sampleCanonicalString = '{"a":{"$numberLong":"23"}}';
+  const sampleRelaxedIntegerString = '{"a":4294967296}';
+  const sampleRelaxedDoubleString = '{"a": 2147483647.9}';
+
+  function genTestTable(
+    useBigInt64: boolean | undefined,
+    relaxed: boolean | undefined,
+    getExpectedResult: (boolean, boolean) => BSON.Document
+  ): [TestTableEntry] {
+    const useBigInt64IsSet = useBigInt64 ?? false;
+    const relaxedIsSet = relaxed ?? true;
+
+    const expectedResult = getExpectedResult(useBigInt64IsSet, relaxedIsSet);
+
+    return [{ options: { useBigInt64, relaxed }, expectedResult }];
+  }
+
+  function generateBehaviourDescription(entry: TestTableEntry, inputString: string): string {
+    // TODO(NODE-4874): When NODE-4873 is merged in, replace this with EJSON.stringify
```

Comment:
This TODO can be addressed now

Reply:
Attempted to use `EJSON.stringify` to help construct test output, but it makes the test output harder to understand, so just going to remove the TODO